website/src/components/subscriptions/overview/SubscriptionEntry.tsx

```diff
@@ -0,0 +1,86 @@
+import type { Subscription } from '../../../types/Subscription.ts';
+import { BorderedCard } from '../../../styles/containers/BorderedCard.tsx';
+import { CardDescription } from '../../../styles/containers/CardDescription.tsx';
+import { SubscriptionDisplay } from './SubscriptionDisplay.tsx';
+
+export function SubscriptionEntry({ subscription }: { subscription: Subscription }) {
+    const getIcon = () => {
+        if (!subscription.active) {
+            return 'mdi--bell-off-outline text-gray-500';
+        }
+
+        if (subscription.triggerEvaluationResult.type === 'ConditionMet') {
+            return 'mdi--bell-ring text-red-500';
+        }
+
+        return `mdi--bell-outline text-gray-500`;
+    };
+
+    return (
+        <BorderedCard>
+            <details className='collapse overflow-visible'>
+                <summary className='collapse-title'>
+                    <div className={`flex items-center justify-between`}>
+                        <CardDescription title={subscription.name} subtitle={'Covid'} icon={`size-6 ${getIcon()}`} />
```

Comment:
The subtitle should be something like `organismConfig[subscription.organism].label` I assume?